include/alpaka/acc/Tag.hpp

```diff
@@ -47,13 +47,13 @@ namespace alpaka
         struct AccToTag;
 
         template<typename TAcc>
-        using AccToTagType = typename AccToTag<TAcc>::type;
+        using Tag = typename AccToTag<TAcc>::type;
 
```

Comment:
Please add a comment to jothbtraits what they should do.

Reply:
As I wrote https://github.com/alpaka-group/alpaka/pull/1804#discussion_r987592586 I think the naming should end with `Type` to fit the already existing traits e.g. `QueueType`

Reply:
Done